src/main/resources/data-mapping.json

```diff
@@ -0,0 +1,201 @@
+{
+  "mappings":
+  [
+    {
+      "field":"ACCOUNT_NUMBER",
+      "dataSource":{
+        "from":"//SubAccount",
+        "default":"0"
+      }
+    },
+    {
+      "field":"ACQUISITION_METHOD",
+      "dataSource":{
+        "default":"mod-gobi"
+      }
+    },
+    {
+      "field":"CLAIM",
+      "dataSource": {
+        "default": "true"
+      }
+    },
+    {
+      "field":"COLLECTION",
+      "dataSource": {
+        "default": "false"
+      }
+    },
+    {
+      "field":"CONTRIBUTOR",
+      "dataSource": {
+        "from": "//datafield[@tag='100']/*",
+        "combinator":"concat"
+      }
+    },
+    {
+      "field":"CREATED_DATE",
+      "dataSource": {
+        "from": "//OrderPlaced"
+      }
+    },
+    {
+      "field":"CURRENCY",
+      "dataSource":{
+        "from":"//NetPrice/Currency",
+        "default":"USD"
+      }
+    },
+    {
+      "field":"EXPECTED_RECEIPT_DATE",
+      "dataSource": {
+        "default": "Order Sent"
```

Comment:
This seems like an odd value for this field, but I'm not as up-to-date with the mappings spreadsheet as you guys